crates/flox/doc/flox-run.md

```diff
@@ -11,21 +11,69 @@ flox-run - run app from current project
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] run [ `<options>` ] [ -- [ `<command args>` ... ] ]
+flox [ `<general-options>` ] run [ `<run-options>` ] [ `<installable>` ] [ -- [ `<command args>` ... ] ]
 
 # DESCRIPTION
 
-Run flake application from the requested package (or "installable").
-If not provided `flox` will prompt for you to select from the list of known packages.
+Run a flake application from the requested installable.
+See the [nix(1)] manual's section on installables for more information.
+
+[nix(1)]: https://nixos.org/manual/nix/stable/command-ref/new-cli/nix.html#installables
+
+# EXAMPLES
+
+## Running applications in the current working directory
+
+If `flox run` is called without any arguments, it will ask the user which installable they want to run.
+Note, in this example, it's assumed there's a `flake.nix` in the current directory.
```

Comment:
```suggestion
If `flox run` is called without any arguments, and a controlling tty is available,
it will offer the user a selection of installable from the current flake.
Note, in this example, it's assumed there's a `flake.nix` in the current directory path.
```